Doxyfile

```diff
@@ -754,7 +754,7 @@ WARN_LOGFILE           =
 # spaces.
 # Note: If this tag is empty the current directory is searched.
 
-INPUT                  = .
+INPUT                  = contrib/epee src
```

Comment:
I'd add easylogging++,

Reply:
This is obviously up for debate, but I decided to not include easylogging++, qrcode, and db_drivers in the documentation since they all have their own separate documentation elsewhere and we would just be redoing their work.

Reply:
easylogging++ is definitely under our maintenance now.

Reply:
Agree with mj. Also, adding links to the docs of the other projects would definitely help.